File: fixtures/r/post_basic_auth_url_encoded_data.r
require(httr)

headers = c(
  `Content-Type` = 'application/x-www-form-urlencoded'
)

data = list(
  `grant_type` = 'client_credentials'
)

res <- httr::POST(url = 'http://localhost/api/oauth/token/', httr::add_headers(.headers=headers), body = data, httr::authenticate('foo', 'bar'))

File: test/fixtures/ruby-httparty/post_basic_auth_url_encoded_data.rb
require 'httparty'

url = 'http://localhost:28139/api/oauth/token/'
auth = { username: 'foo', password: 'bar'}
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = {
  'grant_type' => 'client_credentials'
}
res = HTTParty.post(url, basic_auth: auth, headers: headers, body: body)
